src/capi/misc.cpp

```diff
@@ -15,6 +16,10 @@ void chemfiles::set_last_error(const std::string& message) {
     CAPI_LAST_ERROR = message;
 }
 
+extern "C" void chfl_free(const void* const object) {
```

Comment:
Might make more sense to put this in the shared allocated cpp file, but that's minor.